DevoxxClientMobile/src/main/java/com/devoxx/DevoxxApplication.java

```diff
@@ -158,6 +159,15 @@ public void postInit(Scene scene) {
                 deviceFactorSuffix,
                 formFactorSuffix);
         scene.getStylesheets().add(DevoxxApplication.class.getResource(stylesheetName).toExternalForm());
+
+        String voxxedStylesheet = DevoxxApplication.class.getResource("voxxed.css").toExternalForm();
+        service.conferenceProperty().addListener((observable, oldValue, newValue) -> {
+            if (newValue != null && newValue.getEventType() == Conference.Type.VOXXED) {
```

Comment:
I like to check if the css is already there before it gets added more than once by mistake:
`&&  ! scene.getStylesheets().contains(voxxedStylesheet)) {`... // add
and 
`else if (scene.getStylesheets().contains(voxxedStylesheet)) {`... // remove

Reply:
Good idea! :)

Reply:
In case it doesn't exist, you avoid a call to the rendering engine, nothing that critical at this point of course.